notion2markdown/json2md.py

```diff
@@ -199,7 +221,16 @@ def block_heading(self, value, prv=None, nxt=None):
     @rule
     def block_paragraph(self, value, prv=None, nxt=None):
         if isinstance(value, dict) and value.get("type", "") == "paragraph":
-            return f"{self.json2md(value['paragraph']['rich_text'])}\n"
+            indent = (
+                    (self.config or {}).get("block_item", {}).get("indent", "    ")
+                )  # TODO: make getting config less ugly
+
+            lines = []
+            lines.append(f"{self.json2md(value['paragraph']['rich_text'])}\n")
+            if value["has_children"]:
+                sub = self.jsons2md(value["children"])
+                lines.extend([f"{indent}{line}\n" for line in sub.splitlines()])
```

Comment:
Another good catch, nice.

Re config ugly: Oop, I guess I should initialize config to an empty dictionary. No need to fix that here.

Thanks again!